DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Startup.cs

```diff
@@ -52,16 +52,15 @@ public void Configuration(IAppBuilder app)
 
             app.CreatePerOwinContext<ApplicationUserManager>(ApplicationUserManager.Create);
 
-            FluentValidationModelValidatorProvider
-                .Configure(config,
-                    provider => provider.ValidatorFactory = new FluentValidatorFactory(container));
+            //FluentValidationModelValidatorProvider

```

Comment:
As I know this bug already fixed so please do a rebase on current dev to get latest changes.